libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.BreakLoop.schema

```diff
@@ -20,9 +20,9 @@
             ]
         }
     },
-    "policies": [
+    "$policies": [
         {
```

Comment:
All of these are invalid.  It is not an array.  If you open these in VS code you will see errors.  It should be "$policies": { lastAction: true }

Reply:
oops :/

Thanks @chrimc62 !